src/Components/TabsPage/TabsPage.tsx

```diff
@@ -0,0 +1,42 @@
+import classNames from 'classnames';
+import React from 'react';
+import { Link, useParams } from 'react-router-dom';
+import { Tab } from '../../types/Tab';
+
+type Props = {
+  tabs: Tab[],
+};
+
+export const TabsPage: React.FC<Props> = ({ tabs }) => {
+  const { tabsId = '' } = useParams();
+
+  const selectedTab = tabs.find(tab => tab.id === tabsId) || null;
+
+  return (
+    <div className="section">
+      <div className="container">
+        <div className="tabs is-boxed">
```

Comment:
Betwen this two lines you forget to add ( <h1 className="title">Tabs page</h1> )

Reply:
agree